INFO  | 03/25/2020 01:06:03 PM | File run_exp.py           | Line 26    | Writing log file to exp/GRAN\GRANMixtureBernoulli_transport_2020-Mar-25-13-06-03_21740\log_exp_21740.txt
INFO  | 03/25/2020 01:06:03 PM | File run_exp.py           | Line 27    | Exp instance id = 21740
INFO  | 03/25/2020 01:06:03 PM | File run_exp.py           | Line 28    | Exp comment = None
INFO  | 03/25/2020 01:06:03 PM | File run_exp.py           | Line 29    | Config =
INFO  | 03/25/2020 01:06:04 PM | File gran_runner.py       | Line 124   | Train/val/test = 8/2/2
INFO  | 03/25/2020 01:06:04 PM | File gran_runner.py       | Line 137   | No Edges vs. Edges in training set = 2.0
ERROR | 03/25/2020 01:06:09 PM | File run_exp.py           | Line 42    | Traceback (most recent call last):
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\run_exp.py", line 38, in main
    runner.train()
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\runner\gran_runner.py", line 249, in train
    train_loss = model(*batch_fwd).mean()
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\utils\data_parallel.py", line 104, in forward
    return self.module(*inputs[0], **kwargs[0])
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\model\gran_mixture_bernoulli.py", line 447, in forward
    node_pos=node_pos)
ValueError: too many values to unpack (expected 2)

